spk/salt-master/src/requirements-cross.txt

```diff
@@ -0,0 +1,10 @@
+chardet==3.0.4
+immutables==0.15
+MarkupSafe==1.1.1
+msgpack==1.0.2
+pycryptodomex==3.9.8
+# PyYAML==5.3.1 # included in py3.8
+pyzmq<=20.0.0 ; python_version < "3.6"
+pyzmq>=17.0.0 ; "arm" not in platform_machine
+pyzmq==17.0.0 ; "arm" in platform_machine
+pyzmq>=19.0.2 ; python_version >= "3.9"
```

Comment:
That's interesting.  Also reading on it this could be linked against `cross/libzmq` which might resolve issues found on some platforms.